packages/forklift-console-plugin/src/plans/create/constants.ts

```diff
@@ -0,0 +1,42 @@
+import { t } from 'src/utils/i18n';
+
+export enum PlanWizardStepId {
+  BasicSetUp = 'basic-set-up',
+  General = 'general',
+  VirtualMachines = 'virtual-machines',
+  NetworkMapping = 'network-mapping',
+  StorageMapping = 'storage-mapping',
+  MigrationType = 'migration-type',
+  AdditionalSetUp = 'additional-set-up',
+  OtherSettings = 'other-settings',
+  Hooks = 'hooks',
+  ReviewAndCreate = 'review-and-create',
+}
+
+export const planStepNames: Record<PlanWizardStepId, string> = {
+  [PlanWizardStepId.BasicSetUp]: t('Basic set up'),
+  [PlanWizardStepId.General]: t('General'),
+  [PlanWizardStepId.VirtualMachines]: t('Virtual machines'),
+  [PlanWizardStepId.NetworkMapping]: t('Network mapping'),
+  [PlanWizardStepId.StorageMapping]: t('Storage mapping'),
+  [PlanWizardStepId.MigrationType]: t('Migration type'),
+  [PlanWizardStepId.AdditionalSetUp]: t('Additional set up'),
+  [PlanWizardStepId.OtherSettings]: t('Other settings (optional)'),
+  [PlanWizardStepId.Hooks]: t('Hooks (optional)'),
+  [PlanWizardStepId.ReviewAndCreate]: t('Review and create'),
+};
+
+export const planStepIndexes = Object.values(PlanWizardStepId).reduce(
+  (acc, stepId, index) => ({ ...acc, [stepId]: index + 1 }),
+  {},
+);
+
+export const defaultCurrentStep = {
```

Comment:
I think a more understandable name is firstStepValue? what do u think? like default seems to me like can be used in many places, but this is the actually first step isn't it?

Reply:
That's true. I was considering naming it something similar on the onset, so I'm all for this. I'll just call it `firstStep` and add the type here so it's clear it is of type `WizardStepType`.